content/contributing/documentation/content_guidelines.rst

```diff
@@ -2,228 +2,278 @@
+        select :guilabel:`Page`;
+      - Or go to :menuselection:`Website --> Site --> Pages` and click :guilabel:`New`.
 
-.. _contributing/content-images:
+   #. Enter a :guilabel:`Page Title`; this title is used in the menu and the page's URL.
+   #. Click :guilabel:`Create`.
+   #. Customize the page's content and appearance using the website builder, then click
+      :guilabel:`Save`.
+
+.. seealso::
+   :ref:`RST cheat sheet: lists <contributing/rst/lists>`
+
+.. _contributing/content/images:
 
 Images
 ======
 
-Adding a few images to illustrate your text helps the readers to understand and memorize your
-content. However, avoid adding too many images: it isn't necessary to illustrate all steps and
-features, and it may overload your page.
+Adding a few images to illustrate text helps the readers understand and memorize the content.
+However, images should never replace text: written instructions should be complete and clear on
+their own, without relying on visual aids. Use images sparingly, for example, to highlight a
+particular point or clarify an example.
 
 .. important::
-   Don't forget to :ref:`compress your PNG files with pngquant
+   Do not forget to :ref:`compress your PNG files with pngquant
    <contributing/documentation/first-contribution>`.
 
-.. _contributing/screenshots:
+.. _contributing/content/screenshots:
 
 Screenshots
 -----------
 
-Screenshots are automatically resized to fit the content block's width. This implies that
-screenshots can't be too wide, else they would appear very small on-screen. Therefore, we recommend
-to avoid to take screenshots of a full screen display of the app, unless it is relevant to do so.
+Screenshots are automatically resized to fit the content block's width. This implies that if they
+are too wide, they are not readable on lower-resolution screens. We recommend avoiding full-screen
+screenshots of the app unless absolutely necessary and making sure images are no wider than 768
+pixels.
 
-A few tips to improve your screenshots:
+Here are a few tips to improve your screenshots:
 
-#. **Zoom** in your browser. We recommend a 110% zoom for better results.
 #. **Resize** your browser's width, either by *resizing the window* itself or by opening the
-   *browser's developer tools* (press the ``F12`` key) and resizing the width.
-#. **Select** the relevant area, rather than keeping the full window.
-#. If necessary, you can **edit** the screenshot to remove unnecessary fields and to narrow even
-   more Odoo's display.
+   *browser's developer tools* and resizing the width.
+#. **Select** the relevant area rather than keeping the entire window.
+#. **Remove** unnecessary information and **resize** columns when applicable.
+
+.. important::
+   Do not use markups such as rectangles or arrows on screenshots. Instead, crop the image to
+   highlight the most relevant information and add :ref:`captions <contributing/rst/images>`
+   if absolutely necessary.
```

Comment:
Note for later maybe (I'm sure it's on a to-do list somewhere!)— can add more detailed instructions on **figure** usage? 🙏